src/orange/construct/OrangeInput.hh

```diff
@@ -78,9 +80,17 @@ struct VolumeInput
  */
 struct UnitInput
 {
+    struct Daughter
+    {
+        UniverseId  universe_id;
+        Translation translation;
+    };
+    using Daughter_Map = std::unordered_map<VolumeId, Daughter>;
```

Comment:
I missed the naming issue here: there should be no underscores in type names, and for maps we're so far sticking to `MapKeyValue`. I'll rename this to MapVolumeDaughter.